src/modules/aws-ec2/shutdown-behavior/setShutdownBehaviorToStop.ts

```diff
@@ -0,0 +1,25 @@
+import { changeShutdownBehavior } from "./sub-modules/changeShutdownBehavior";
+import { RunbookStep } from "../../RunbookStep";
+import { logger } from "../../../logger";
+import { describeShutdownProtection } from "./sub-modules/describeShutdownBehavior";
+
+export class SetShutdownBehaviorToStop extends RunbookStep {
+  async describeAction(instanceId: string): Promise<void> {
+    const currentValue = await describeShutdownProtection(instanceId);
+    const targetValue = "stop";
```

Comment:
Maybe this should be moved to outside this function since both `describeAction` and `run` methods use it?